Resources/public/Admin.js

```diff
@@ -74,7 +74,8 @@ var Admin = {
 
                 select.select2({
                     width: function(){
-                        return Admin.get_select2_width(this.element);
+                        // Select2 v3 and v4 BC. If window.Select2 is defined, then the v3 is installed.
```

Comment:
I don't see what to say on the upgrade note. The Select2 v3 is not really deprecated as always provided by default.

This is just an addition to **permit** user to upgrade manually.